tutorial/creating_packages/other_types_of_packages/header_only_packages.rst

```diff
@@ -44,6 +44,15 @@ This is a basic recipe for a header-only recipe:
             # This will also copy the "include" folder
             copy(self, "*.h", self.source_folder, self.package_folder)
 
+        def package_info(self):
+            # For header-only packages, libdirs and bindirs are not used
+            # so it's recommended to set those as empty.
+            self.cpp_info.bindirs = []
+            self.cpp_info.libdirs = []
+
+Please, note that we are setting cpp_info.bindirs and cpp_info.libdirs to ``[]`` because
+we won't use them for header-only.
```

Comment:
```suggestion
Please, note that we are setting ``cpp_info.bindirs`` and ``cpp_info.libdirs`` to ``[]`` because
header-only libraries don't have compiled libraries or binaries, but they default to ``["bin"]``, and ``["lib"]``, then it is necessary to change it.
```